src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -398,7 +399,7 @@ export default class OncoprintControls extends React.Component<IOncoprintControl
                 return <span/>;
             } else {
                 menu = (
-                    <div style={{padding: "10px 5px", width: "270px", minWidth: "270px"}}>
+                    <div className="oncoprint__controls__heatmap_menu">
```

Comment:
whats going on with this? should this be something like styles['oncoprint_...']

Reply:
I think in many cases css module approach is unnecessary.  I'm using a naming convention called BEM to avoid conflict.

Reply:
ah I see